Dockerfile

```diff
@@ -35,6 +34,7 @@ ENV PATH="${N64_INST}/bin:$PATH"
 
 COPY --from=0 ${N64_INST} ${N64_INST}
 RUN apt-get update && \
-    apt-get install -yq gcc g++ make libpng-dev git  && \
+    apt-get install -yq gcc g++ make libpng-dev git && \
+    apt-get install -yq ninja-build rsync zip && \
```

Comment:
Why are you adding these packages in the final docker image?

Reply:
This should enable compiling ROM's using CMAKE (yet to test) and also starts work on being able to use as a `dev container` within VSCode. Although this should (probably) be a seperate PR... I will split this out when I get everything else working, as this PR is now a bit of a mess... But at least shows progress in the grand scheme of things!